src/main/java/com/mongodb/jdbc/MongoDriver.java

```diff
@@ -635,8 +637,8 @@ private static String buildNewURI(
             Properties options)
             throws SQLException {
         // The returned URI should be of the following format:
-        //"mongodb://[user:password@]host1[:port1][,host2[:port2],...[,hostN[:portN]]][/[authDatabase][?options]]")
-        String ret = "mongodb://";
+        //"mongodb(+srv)?://[user:password@]host1[:port1][,host2[:port2],...[,hostN[:portN]]][/[authDatabase][?options]]")
+        String ret = isSrvProtocol ? "mongodb+srv://" : "mongodb://";
```

Comment:
This was also something that caused a delay! Without this, we were hitting an unrelated timeout error. This doesn't actually enable Atlas support, of course, since we still attempt to use sql commands. This just ensures the error we get back now is "command not found: sqlGetResultSchema" as opposed to a timeout.